src/containers/App.js

```diff
@@ -3,9 +3,12 @@ import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Home from '../components/Home';
 import NotFound from '../components/NotFound';
 import Login from '../components/Login';
+import Auth from './Auth';
 
 const App = () => (
+
   <Router>
+    <Auth />
```

Comment:
Good job so far @innocentEdosa, however, kindly add this `<Auth  />` to the `switch block` as the current implementation is calling the auth modal in every route.

Reply:
It's currently outside the switch block because we want to have the auth modal on every route. the auth modal display property has false by default  and so would not show until an event warrants it